packages/core/admin/admin/src/pages/MarketplacePage/index.js

```diff
@@ -242,9 +244,9 @@ const MarketPlacePage = () => {
                 </Tab>
               </Tabs>
             </Box>
-            <Box paddingBottom={4}>
-              <SortFilter />
-            </Box>
+            <Flex paddingBottom={4} gap={2}>
+              <SortFilter value={sort} onChange={(sort) => setQuery({ ...query, sort })} />
```

Comment:
You don't need to do `...query` here, that logic is already [handled](https://github.com/strapi/strapi/blob/359eeabc5b9ac3e93401a192767265d93e388dca/packages/core/helper-plugin/lib/src/hooks/useQueryParams/index.js#L28) by useQueryParams